### ISSUE TITLE
Documentation coverage

### ISSUE THREAD

**Issue by cope** [senchalabs/jsduck#666]
Any plans on adding documentation coverage reports, like [this one from esdoc](https://doc.esdoc.org/github.com/esdoc/esdoc/source.html)?

'Cause that would be awesome!

**Comment by nene** [comment]
The truth is that JSDuck is not really maintained any more. So less than
unlikely to happen.

28.02.2017 19:39 kirjutas kuupäeval "Predrag Stojadinović" <
notifications@github.com>:

**Comment by michaelkantor** [comment]
Is there a successor?  Is there something different that Sencha is now recommending?  I've been using this for 3 years and remain pretty happy with it, but always happy to look at other projects.

**Comment by cope** [comment]
What is Sencha using for documentation now?

**Comment by nene** [comment]
My guess is that they're still using JSDuck under the hood to extract docs from source code, but have built a new UI for showing the docs.

But these are just guesses. I'm not in contact with Sencha any more.

**Comment by valioDOTch** [comment]
It appears this forum thread is related to this question
https://www.sencha.com/forum/showthread.php?311491

> greg.barry: I'd love for the new docs software to be utilized by the community. That said, it's not currently set up to be shared outside of Sencha yet. We'll have to have some internal dialogue about the best way to potentially distribute this for other developers to use.

**Comment by valioDOTch** [comment]
PS: it appears there is some undocumented tool available via SenchaCmd called doxi
https://www.sencha.com/forum/showthread.php?311491-Will-the-new-documentor-be-available-for-our-own-apps&p=1137925&viewfull=1#post1137925
figuring out the details now for the config file...
[project.doxi.json.zip](https://github.com/senchalabs/jsduck/files/838839/project.doxi.json.zip)

**Comment by michaelkantor** [comment]
Yeah; I saw that. However, it only outputs JSON, and leaves it to you to build the HTML.  I rather like the JSDuck HTML; its part of why I use it (I'm not developing using a sencha JS framework).  I don't currently have time budgeted to design and build an entire HTML generator that is as good.  This project still works; hopefully they'll have open sourced everything before this starts to break down.

**Comment by valioDOTch** [comment]
The problem I have with JSDuck currently is that there's no new release which would fix some bugs like #491 :-(
